File: 0009-palindrome-number/0009-palindrome-number.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        int org  = x ;
        long rev = 0 ;
        while( x > 0 ){
            int digit = x % 10 ;
            x = x /10;
            rev = rev * 10  + digit ;
        }
        if (org == rev){
            return  true;
        }
        else{
            return false;
        }
    }
};

File: 0009-palindrome-number/0009-palindrome-number.py
class Solution:
    def isPalindrome(self, x: int) -> bool:
        org  = x ;
        rev = 0 ;
        while x > 0 :
            digit = x % 10 
            x = x //10
            rev = rev * 10  + digit 
        
        if org == rev :
            return  True
        
        else :
            return False
        